eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/ssz_static/SszTestExecutor.java

```diff
@@ -0,0 +1,131 @@
+import tech.pegasys.teku.ssz.sos.SimpleOffsetSerializable;
+import tech.pegasys.teku.util.hashtree.Merkleizable;
+
+public class SszTestExecutor<T extends SimpleOffsetSerializable & Merkleizable>
+    implements TestExecutor {
+  private final Class<T> clazz;
+
+  public static ImmutableMap<String, TestExecutor> SSZ_TEST_TYPES =
+      ImmutableMap.<String, TestExecutor>builder()
+          // SSZ Static
+          .put("ssz_static/AggregateAndProof", new SszTestExecutor<>(AggregateAndProof.class))
+          .put("ssz_static/Attestation", new SszTestExecutor<>(Attestation.class))
+          .put("ssz_static/AttestationData", new SszTestExecutor<>(AttestationData.class))
+          .put("ssz_static/AttesterSlashing", new SszTestExecutor<>(AttesterSlashing.class))
+          .put("ssz_static/BeaconBlock", new SszTestExecutor<>(BeaconBlock.class))
+          .put("ssz_static/BeaconBlockBody", new SszTestExecutor<>(BeaconBlockBody.class))
+          .put("ssz_static/BeaconBlockHeader", new SszTestExecutor<>(BeaconBlockHeader.class))
+          .put("ssz_static/BeaconState", new SszTestExecutor<>(BeaconStateImpl.class))
+          .put("ssz_static/Checkpoint", new SszTestExecutor<>(Checkpoint.class))
+          .put("ssz_static/Deposit", new SszTestExecutor<>(Deposit.class))
+          .put("ssz_static/DepositData", new SszTestExecutor<>(DepositData.class))
+          .put("ssz_static/DepositMessage", new SszTestExecutor<>(DepositMessage.class))
+          .put("ssz_static/Eth1Block", IGNORE_TESTS) // We don't have an Eth1Block structure
+          .put("ssz_static/Eth1Data", new SszTestExecutor<>(Eth1Data.class))
+          .put("ssz_static/Fork", new SszTestExecutor<>(Fork.class))
+          .put("ssz_static/ForkData", new SszTestExecutor<>(ForkData.class))
+          .put("ssz_static/HistoricalBatch", new SszTestExecutor<>(HistoricalBatch.class))
+          .put("ssz_static/IndexedAttestation", new SszTestExecutor<>(IndexedAttestation.class))
+          .put("ssz_static/PendingAttestation", new SszTestExecutor<>(PendingAttestation.class))
+          .put("ssz_static/ProposerSlashing", new SszTestExecutor<>(ProposerSlashing.class))
+          .put(
+              "ssz_static/SignedAggregateAndProof",
+              new SszTestExecutor<>(SignedAggregateAndProof.class))
+          .put("ssz_static/SignedBeaconBlock", new SszTestExecutor<>(SignedBeaconBlock.class))
+          .put(
+              "ssz_static/SignedBeaconBlockHeader",
+              new SszTestExecutor<>(SignedBeaconBlockHeader.class))
+          .put("ssz_static/SignedVoluntaryExit", new SszTestExecutor<>(SignedVoluntaryExit.class))
+          .put("ssz_static/SigningRoot", IGNORE_TESTS) // TODO: Should make this work
+          .put("ssz_static/Validator", new SszTestExecutor<>(Validator.class))
+          .put("ssz_static/VoluntaryExit", new SszTestExecutor<>(VoluntaryExit.class))
+
+          // SSZ Generic
+          .put("ssz_generic/basic_vector", IGNORE_TESTS)
```

Comment:
Why are we ignoring these tests?

Reply:
We've never run the ssz_generic tests previously.  Would be good to enable at least some of them like bitvector and bitlist which should be a lot simpler now.